Formula/jolie.rb

```diff
@@ -0,0 +1,50 @@
+class Jolie < Formula
+  desc "The Jolie Language Interpreter"
+  homepage "https://www.jolie-lang.org/"
+  url "https://github.com/jolie/jolie/releases/download/v1.9.0/jolie-1.9.0.jar"
+  sha256 "1510ed7f114909eb79670462571ab0734a5b01e57d26da6fd1cf9ef6c67eff6e"
+
+  depends_on "openjdk"
+
+  def install
+    system "java", "-jar", "jolie-#{version}.jar", "--jolie-home", libexec, "--jolie-launchers", libexec/"bin"
```

Comment:
Now that I'm reading it again, this would probably be even better
```suggestion
    system Formula["openjdk"].opt_bin/"java", "-jar", "jolie-#{version}.jar", "--jolie-home", libexec, "--jolie-launchers", libexec/"bin"
```

Reply:
The audit had some problems with the length of the command.
Fixed.